CodeStarterCamp_Week2/LottoWinningChecker.swift

```diff
@@ -29,3 +30,23 @@ func checkLottoNumber(compare myLotto: Array<Int>, with winningNumbers: Set<Int>
         print("축하합니다! 겹치는 번호는 \(checkedNumbersToString) 입니다!")
     }
 }
+
+// STEP - 3
+// 로또 회차를 5번 생성한 후, 회차별로 Dictionry에 저장
+func makeFiveTimesLotto() -> [String: [Int]] {
+    var fiveTimesLottoNumbers: [String: [Int]] = [:]
+    let LottoNumberStore: [Int] = Array(makeLottoNumber())
+    
+    for round in 1...5 {
+        fiveTimesLottoNumbers.updateValue(LottoNumberStore, forKey: "\(round)회차")
+    }
+    return fiveTimesLottoNumbers
+}
+
+// 특정 회차 로또 당첨 번호 확인
+func readLottoNumbers(when round: Int, from wholeTimesLottoNumbers: [String: [Int]]) {
```

Comment:
호출할 때 when 뒤에 바로 정수가 들어가면 어떤 파라미터인지 알 수 없을 것 같아요.
이 경우에는 전달인자 레이블을 없애도 괜찮아 보입니다.
```suggestion
func readLottoNumbers(from wholeTimesLottoNumbers: [String: [Int]], round: Int) {
```